projects/common/src/utilities/formatters/string/string-formatter.ts

```diff
@@ -1,5 +1,7 @@
 import { isEmpty } from 'lodash-es';
 
+export const titleCase = (str: string): string => (str.length === 0 ? str : str[0].toUpperCase() + str.slice(1));
```

Comment:
lodash has a method for this. Also there is a pipe `titleCase` from angular. Do we need our own?